File: 1. Basics Of Programming/1.4 primeTillN/main.cpp
#include<iostream>
using namespace std;
bool isPrime(int n){
    if(n == 0 || n == 1){
        return false;
    }
    for(int fac = 2 ; fac * fac <= n ; fac++){
        if(n%fac == 0){
            return false;
        }
    }
    return true;
}
void printAllPrimes(int low,int high){
    while(low <= high){
        if(isPrime(low)){
            cout<<low<<endl;
        }
        low++;
    }
}
int main(){
    int low ,high;
    cin>>low>>high;
    printAllPrimes(low , high);
}

File: 1. Basics Of Programming/1.4 primeTillN/Main.java
import java.util.Scanner;
public class Main{
    public static void main(String args[]){
        Scanner scn = new Scanner(System.in);
        int low = scn.nextInt();
        int high = scn.nextInt();

        printAllPrimes(low,high);
    }
    public static void printAllPrimes(int low , int high){
        for(int n = low ; n <= high ; n++){
            if(isPrime(n))
                System.out.println(n);
        }
    }

    public static boolean isPrime(int n){
        if(n == 0 || n == 1){
            return false;
        }
        for(int fac = 2 ; fac * fac <= n ; fac++){
            if(n % fac == 0){
                return false;
            }
        }
        return true;
    }
}